internal/controller/server/execution/execution_test.go

```diff
@@ -2,18 +2,21 @@ package execution
 
 import (
 	"context"
+	"errors"
+	"github.com/stretchr/testify/assert"
+	"octavius/internal/controller/server/repository/executor"
+	job "octavius/internal/controller/server/repository/job"
 	"octavius/internal/controller/server/repository/metadata"
-	"octavius/internal/pkg/log"
-	protobuf "octavius/internal/pkg/protofiles/client_cp"
-	"reflect"
+	"octavius/internal/controller/server/scheduler"
+	"octavius/internal/pkg/idgen"
 	"testing"
 )
 
 func init() {
-	log.Init("info", "")
+	//log.Init("info", "")
```

Comment:
you should not comment log.Init here and pass in the three parameters needed by the function